pyuploadcare/conf.py

```diff
@@ -29,3 +29,8 @@
 retry_throttled = 1
 
 user_agent_extension = None
+
+# expire period for signed URL uploads, in seconds
+window_period = None
+# can be 'akamai' or 'keycdn'
+cdn_provider = None
```

Comment:
```suggestion
cdn_provider = 'akamai'
```

also, see the comment about `conf.sign_urls`.